File: VB/DataGridWinApp1/DataClass.vb
Imports System

Namespace DataGridWinApp1

    Public Class DataClass

        Public Sub New(ByVal seed As Integer)
            IntProperty = seed
            StringProperty = "Text " & seed
            DateProperty = DateTime.Now.AddDays(-seed)
        End Sub

        Public Sub New()
        End Sub

        Public Property IntProperty As Integer

        Public Property StringProperty As String

        Public Property DateProperty As DateTime
    End Class
End Namespace

File: CS/DataGridWinApp1/DataClass.cs
using System;

namespace DataGridWinApp1
{
	public class DataClass
	{
		public DataClass(int seed)
		{
			IntProperty = seed;
			StringProperty = "Text " + seed;
			DateProperty = DateTime.Now.AddDays(-seed);
		}

		public DataClass()
		{
		}

		public int IntProperty
		{
			get;
			set;
		}

		public string StringProperty
		{
			get;
			set;
		}

		public DateTime DateProperty
		{
			get;
			set;
		}
	}
}
